### DISCUSSION TITLE
CAD not working with FF 86

### DISCUSSION THREAD

**Discussion by akasaka99** [Cookie-AutoDelete/Cookie-AutoDelete#1062] [Q&A / Support] [answered]
Hi -
With the new release of Firefox 86, Auto-Delete Cookies does not work anymore.
1) Closing a Tab does not clear cookies which was working fine before
2) Forcing to clear cookies (clicking Clean) does not work either
3) Pretty much everything stopped working
Will it work with the new FF's Total Cookie Protection or there are some new setting that need to be checked?
Many thanks

**Comment by kennethtran93** [comment]
Looks like it does remove some cookies but not all.  I need to find time to rework the notification to actually show whether or not cookies were deleted as we do not check if it is actually removed at this time.

**Reply by kennethtran93** [reply]
As for as the cookie removal functions go, I don't see anything different on the public documentation itself at this time.  I'll need to dig deeper later.

**Reply by Gene-WA** [reply]
I'm having a similar issue with CAD on Firefox 86. Briefly, I run the Clean routine, and if I run it again soon after, a lot of the same cookies are still being found but not deleted. This might've started with Firefox 86, I've been noticing it for a week or two. One example - Google keeps showing up in the list, even though I have it neither White or Gray listed, so it should always be deleted. But it's not.

This morning I used Firefox to delete all cookies, but found that CAD was behaving the same way later in the day. Let me know if there's something specific I should try (also, my first post on Github).

**Reply by akasaka99** [reply]
Yes absolutely its FF 86. I have the same issue which started exactly with FF 86. 
Furthermore, forcing the manual cookie cleaning of any site or by domain does not work either.Cleaning a tab while closing it does not work. So I am not sure it is a definite structural issue with the extension which rendered it obsolete or they need to update.

**Reply by akasaka99** [reply]
It would be great if the dev. team could keep us to date with the situation: is it a definite issue or something resolvable.

**Comment by kennethtran93** [comment, marked as answer]
I cannot say for sure at this time, while we are able to remove some cookies, there have been outstanding cookies that even other webextensions like Cookie Quick Manager and Cookie Manager cannot remove either. This most likely ties into the new tracking protection, and will need some time to figure out how it works as I don't see too many documentations on how webextensions can handle those at this time.

#807 was the first issue way back when it was in Nightly. It's currently pinned on the issues area. Probably best to keep track of it there.

This conversation is now locked, and comments relating to this should be put into 807 above. They've disabled conversions from discussion to issues temporarily so I cannot move and close this at this time.
